assets/sass/_container.scss

```diff
@@ -0,0 +1,33 @@
+.auth-container {
+  border-radius: 3px;
+  border-style: solid;
+  border-width: 1px;
+  height: auto;
+  margin: 100px auto 8px;
+  width: $container-width;
+  min-width: $container-min-width;
+  @include box-sizing(border-box);
+
+  @include device-mq(x-small) {
+    margin-top: 0;
+    width: 100%;
+  }
+
+  @include max-width-mq($container-width) {
+    width: 100%;
+  }
+
+  @include max-height-mq(750px) {
+    margin-top: 0;
+  }
+}
+
+.no-beacon .auth-content {
```

Comment:
In this case, we know we want to use an id - let's just disable that lint warning in this file and the other one where we'll use the id's